### ISSUE TITLE
[Task] Vuetify 3 Release Checklist

### ISSUE THREAD

**Issue by johnleider** [vuetifyjs/vuetify#14984]
# V3 Release Checklist

The following is an in progress list of remaining tasks before we release the next version. 

This will be updated daily; so watch for changes.

Discord, https://community.vuetifyjs.com/ and join channel **#v3-discussion**.

## Final review

- [x] Framework Core #15139
- [x] Documentation
- [x] UI Components
- [x] Tooling

### Documentation

- [x] Introduction
- [x] Getting started
- [x] Features
- [x] Styles and animations
- [x] Directives
- [x] Resources
- [x] About

### Tooling

- [x] CLI plugin
- [x] Vite plugin
- [x] Presets
- [x] Eslint

### UI Components

A component is considered complete when the following criteria is met:

<details>

* Primary and child components reviewed
  * imports ordered with appropriate comments
  * converted to use grid css structure (if applicable)
  * unit tests
    * jest spec test (if applicable)
    * cypress e2e test
* SASS/SCSS code reviewed
  * removes deprecated or unused code
  * abstracted explicit values to variables
  * verified sass variable usage
* index.ts file export formatted
* Documentation page
  * all examples working with no deprecated code
  * removed unused or deprecated examples
  * added **all components** page image
  * added quick bar and links
  * updated page entry and formatting
  * new sections added
    * accessibility (if applicable)
    * anatomy
    * grid (if applicable)
    * sass variables
    * theme (if applicable)
  * updated api section formatting
  * frontmatter information reviewed
</details>

- [x] v-alert #14971
- [x] v-app #15179
- [x] v-app-bar #15192
- [x] v-autocomplete #15355
- [x] v-avatar #15112
- [x] v-badge #15193
- [x] v-banner #15109
- [x] v-bottom-navigation #15194
- [x] v-breadcrumbs #15181
- [x] v-btn #15213
- [x] v-btn-group #15195
- [x] v-btn-toggle #15196
- [x] v-card #15283
- [x] v-carousel #15197
- [x] v-checkbox #15264
- [x] v-chip #14973
- [x] v-chip-group #15198
- [x] v-code #15182
- [x] v-color-picker #15292
- [x] v-combobox #15354
- [x] v-counter #15183
- [x] v-default-provider
- [x] v-dialog #15289
- [x] v-divider #15113
- [x] v-expansion-panels #15295
- [x] v-field #15251
- [x] v-file-input #15288
- [x] v-footer #15114
- [x] v-form #15253
- [x] v-grid #15296
- [x] v-hover
- [x] v-icon #15212
- [x] v-img #15287
- [x] v-input #15260
- [x] v-kbd #15199
- [x] v-label #15256
- [x] v-lazy #15184
- [x] v-list #15360
- [x] v-locale-provider #15255
- [x] v-main #15254
- [x] v-menu 5e4992ffa
- [x] v-messages #15200
- [x] v-navigation-drawer #15263
- [x] v-no-ssr #15185
- [x] v-overlay #15284
- [x] v-pagination #15286
- [x] v-parallax #15226
- [x] v-progress-circular #15266
- [x] v-progress-linear #15265
- [x] v-radio #15258
- [x] v-radio-group #15259
- [x] v-range-slider #15228
- [x] v-rating #15285
- [x] v-responsive #15249
- [x] v-selection-control #15246
- [x] v-selection-control-group #15250
- [x] v-sheet #15115
- [x] v-slide-group #15248
- [x] v-slider #15227
- [x] v-snackbar #15242
- [x] v-switch #15247
- [x] v-system-bar #15243
- [x] v-table #15186
- [x] v-tabs #15262
- [x] v-text-field #15252
- [x] v-textarea #15257
- [x] v-theme-provider
- [x] v-timeline #15230
- [x] v-toolbar #15154
- [x] v-tooltip #15245
- [x] v-validation
- [x] v-window #15244

**Comment by michaelnguyen08** [comment]
v-time-picker and v-date-picker not coming with this release?

**Comment by omerkimel** [comment]
What about v-data-table?

**Comment by JesusFregoso** [comment]
What about v-skeleton-loader ?

**Comment by gregveres** [comment]
@michaelnguyen08, @omerkimel , @JesusFregoso You can see the list of components that didn't make the 3.0 cut in the [3.1 milestone](https://github.com/vuetifyjs/vuetify/milestone/56).

These were cut from the initial release fairly early this year. Noteably this includes the following components:

- Calendar
- TimePicker
- data-iterator
- date picker
- v-stepper
- v-speed-dial
- v-skeleton-loader
There are some feature requests in the milestone that I am hoping John and the team will punt to a later release to expedite getting these base components released. 


v-data-table was mentioned in above, but that is supposed to be in v3.0.0 milestone. I just looked at the docs for v3 and it looks like v-data-table was renamed to v-table. v-table is in the docs and in the list above so that appears to have made the cut for v3.0.

**Comment by AquaMCU** [comment]
Hi. I just opened the node_modules/vuetify/lib/components directory and I can see, that a component directory "VDataTable" is included in the Beta.1 build. When I try to include it in my App, I see, that VUE is not able to resolve it "[Vue warn]: Failed to resolve component: v-data-table". Seems, that the component is just not registered. 

And I disagree, that will be renamed...

**Comment by MajesticPotatoe** [comment]
> Hi. I just opened the node_modules/vuetify/lib/components directory and I can see, that a component directory "VDataTable" is included in the Beta.1 build. When I try to include it in my App, I see, that VUE is not able to resolve it "[Vue warn]: Failed to resolve component: v-data-table". Seems, that the component is just not registered.

Directory is still there because its not that it is being removed, it just hasn't been converted.
this: https://github.com/vuetifyjs/vuetify/blob/next/packages/vuetify/src/components/index.ts
controls what actually get exported and is what is currently available to use

**Comment by zorn-v** [comment]
Where is `loading` prop in VBtn component ? 
Is it removed completely or added later ?
It was amazing button feature.

**Comment by aentwist** [comment]
I am guessing that not only are components being cut, but also props? For example, Menu is missing a fantastic amount of props functionality -

https://vuetifyjs.com/en/api/v-menu/#props
https://next.vuetifyjs.com/en/api/v-menu/#props

I need `left` 😕

What is the plan with this?

**Comment by warflash** [comment]
> I am guessing that not only are components being cut, but also props? For example, Menu is missing a fantastic amount of props functionality -
> 
> https://vuetifyjs.com/en/api/v-menu/#props https://next.vuetifyjs.com/en/api/v-menu/#props
> 
> I need `left` 😕
> 
> What is the plan with this?

@aentwist 
Well v-menu is still WIP, you can track the progress here: https://github.com/vuetifyjs/vuetify/issues/13489 or in the [3.0.0 milestone](https://github.com/vuetifyjs/vuetify/milestone/45)

> I need `left` 😕

I have not used v3 yet but based on the docs it seems like `:anchor="start"` could be what you are looking for?

**Comment by johnleider** [comment]
> I am guessing that not only are components being cut, but also props? For example, Menu is missing a fantastic amount of props functionality -
> 
> 
> 
> https://vuetifyjs.com/en/api/v-menu/#props
> 
> https://next.vuetifyjs.com/en/api/v-menu/#props
> 
> 
> 
> I need `left` 😕
> 
> 
> 
> What is the plan with this?

I don't think we have any completely removed props. Some may have been combined to be more intuitive, but they are there.

In regards to v-btn, the v-progress-circular component wasn't ported when it was. It'll be there, I just have to get to it.

**Comment by aentwist** [comment]
I have a feeling I am imagining the combination of `offset-y` and `left`. Thanks for the tip with `anchor`. Thanks to both for the info about the current state of the props.

**Comment by Stevelriemenbill** [comment]
> What about v-data-table?

see following answer for those interested: https://github.com/vuetifyjs/vuetify/issues/13479#issuecomment-1115233462

**Comment by MatthewAry** [comment]
Would like to suggest adding a Nuxt3 plugin to this list as well.

**Comment by warflash** [comment]
> Would like to suggest adding a Nuxt3 plugin to this list as well.

@MatthewAry 
I'd love to see first class nuxt support as well, though that's something done by [nuxt ](https://github.com/nuxt-community/vuetify-module) itself, not vuetify.
Related: 
https://github.com/vuetifyjs/vuetify/issues/14621#issuecomment-1016186592
https://github.com/nuxt-community/vuetify-module/issues/475

**Comment by MatthewAry** [comment]
@warflash If that's what @KaelWD says then I suppose that's how it is. However I am really anxious to see support for SASS Variables in Nuxt3 for the Vuetify project.

**Comment by Azema4ka** [comment]
> Где находится `loading`поддержка в компоненте VBtn? Он удален полностью или добавлен позже? Это была удивительная функция кнопки.

https://github.com/vuetifyjs/vuetify/issues/15056

**Comment by edmundmunday** [comment]
For those components not included in the initial release, is there an incremental path for people who's apps use those components, or are they just going to have to wait until 3.1 before they can upgrade?

**Comment by MartinX3** [comment]
Isn't 
3.0
3.1
3.2
3.x
incremental enough?

Just wait until everything is proper released.
I can't understand people needing to upgrade to a new version as soon as possible, even if it is Beta.

The only thing I would like would be a comparison table about which component gets added into which version.

Also you can wait for Vue 2.7 and Vueitfy 2.7.
These are already announced migration versions with backported features.....

**Comment by gregveres** [comment]
> For those components not included in the initial release, is there an incremental path for people who's apps use those components, or are they just going to have to wait until 3.1 before they can upgrade?

If you rely on some of the components that aren't going to be in 3.0, I see two choices:

1. you wait until the release that contains all of the components you use
2. you replace the missing components with components you build

I have been waiting for Vuetify 3 so that I can migrate to Vue 3 and Vite, but I have chosen option 1 above.  I use most of the components that aren't available in 3.0

- Data Table
- Calendar
- TimePicker
- date picker
- v-stepper
- v-skeleton-loader

**Comment by websitevirtuoso** [comment]
Please add v-data-table in this release. Thanks

**Comment by johnleider** [comment]
> Please add v-data-table in this release. Thanks

How? It's not done.

**Comment by sinisarudan** [comment]
> @MartinX3 I can't understand people needing to upgrade to a new version as soon as possible, even if it is Beta.
I see that you don't understand, but the point is if you start a new project, you DON'T want to start it in v.2.x and then to migrate it all to v.3.x. Take in mind that it is not only vuetify 3, but you are by being forced to use vuetify 2, forced to use vue 2 too! So it is pretty pretty painful enforcement!

Thus, developers and PMs are "enforced" and motivated to do additional work and workarounds etc to enable v.3 version working, then to have the enormous work later in migrating whole the project.

I hope that collaborators on Vuetify are aware of this too :)

**Comment by MartinX3** [comment]
@sinisarudan I see you prefer very much time and energy into complaining instead and also investing very much time and energy to do workarounds and extra stuff, just to use as soon as possible the newest not ready software.
Weird people. I hope you won't become my employee.

smart people just use vue 2 with vuetify 2 and just add the composition-api package and use typescript.
Also smart people know that according to the roadmap you can just wait for vue 2.7 and vuetify 2.7 to easily migrate your project on 3.x without much work.
(I use vue/vuetify as frontend for customers and also nuxtjs for (static) websites)

**Comment by sinisarudan** [comment]
> @MartinX3
I too hope you will never become a consultant for my businesses or for any of my partners.
They as I don't like those that follow **Vuetify 3 checklist** issue to complain about people giving advices to others and discussing workarounds. Just relax, man, and let others have different opinion that you.
In the whole thread this long, you're the first one bringing so much negative energy in a single comment.

**Comment by km2442** [comment]
I am not sure of anything anymore, I have read a lot of contradictory information... Will v-data-table be available in 3.0 or do I have to wait for 3.1?

**Comment by abumalekFayed** [comment]
In vuetify 3 documentation roadmap current release is alpha please correct this to avoide misconsiption
![Screenshot 2022-05-15 145002](https://user-images.githubusercontent.com/47514686/168473783-cfb08ef3-6b58-42c8-a94b-a98de7e55418.png)
.

**Comment by MartinX3** [comment]
?
![grafik](https://user-images.githubusercontent.com/4518092/168476170-0fa27988-444d-4821-9e3c-12510dad7538.png)

**Comment by abumalekFayed** [comment]
i mean in this link 
https://next.vuetifyjs.com/en/introduction/roadmap/

**Comment by MartinX3** [comment]
`This is BETA documentation for Vuetify 3, examples and information may be broken or outdated.`

**Comment by AndreyYolkin** [comment]
> I am not sure of anything anymore, I have read a lot of contradictory information... Will v-data-table be available in 3.0 or do I have to wait for 3.1?

It's not listed in checklist, so we need to wait for 3.1

**Comment by eliezerp3** [comment]
Target release is this month. Is this checklist actually only 1/90 or have they just not been updating it?

**Comment by MartinX3** [comment]
@eliezerp3 if you want to avoid a delay please submit PR's to port the components.

**Comment by eliezerp3** [comment]
@MartinX3 Not looking for a fight. It was a genuine question…

**Comment by MartinX3** [comment]
> @MartinX3 Not looking for a fight. It was a genuine question…

I'm not looking for a fight.
Just the issue ticket gets updated daily if something changed. :P

**Comment by johnleider** [comment]
I'm working on getting the tuning pass PRs up. Some will be involved reviews, over half are simple and will not.

**Comment by zorn-v** [comment]
@johnleider
Please check https://github.com/vuetifyjs/vuetify/pull/15132

**Comment by Tan-Jerry** [comment]
when can I use the vuetify 3.0.

**Comment by throrin19** [comment]
> In regards to v-btn, the v-progress-circular component wasn't ported when it was. It'll be there, I just have to get to it.

@johnleider but the `v-progress-circular` component is in the beta 2

**Comment by zorn-v** [comment]
> @Tan-Jerry after you submit PR's to accelerate porting.

Did you try it by yourself ? 
PR's does not accelerate anything. Just for example
https://github.com/vuetifyjs/vuetify/pull/15132
https://github.com/vuetifyjs/vuetify/pull/15129

**Comment by MartinX3** [comment]
> Well, just edit your comment to
> 
> ```diff
> - you submit PR's
> + donate money
> ```

@zorn-v seeing ghosts is a dangerous mental health constitution

**Comment by zorn-v** [comment]
No, I have "take this" sword.
But you are fat troll and I do not want to blunt the sword

**Comment by johnleider** [comment]
Keep it civil please. We are going to get to everything, including community submitted PRs.

**Comment by sinisarudan** [comment]
Hmm, I was trying to keep bearing with Vuetify 3 (even implementing some workarounds on our ongoing projects) until target release is here.
Yet:

- official deadline for Vuetify Target release has passed (May) but we're still in Beta
- there are no answers from developers regarding not-working functionality even the one documented in Beta as working https://github.com/vuetifyjs/vuetify/discussions/15054
- we do not get updates when something is going to be delivered, after the self-defined deadlined has passed, so we can hardly make valid decisions
- as some of you mentioned here, it seems that the TARGET release of 3.0 (expected in the passed May) won't be a "real" target release in sense it will still lack too many components of their functionality, compared to 2.x version ...
- I would like to hear some opposing reassuring thoughts from developers, yet it seems that the waiting and workarounds inspired by "too optimistic" announcements by Vuetify developers is not justified anymore :(
- Especially that this is not ony waiting for Vuetify 3 but also preventing us from switching from Vue2 to Vue 3 and all the new ecosystem being built around it (so waiting on 2.7 and suggested (automatic) migration to 3.x using it is neither solution, esp. that it has also been prolonged for several months)

In another thread https://github.com/vuetifyjs/vuetify/discussions/14776 the [Quasar](https://quasar.dev/) is mentioned as an alternative while we wait for the Vuetify 3 reaching that level of stability and versatility. I guess I will give it a try.

P.S. Dear @MartinX3 please keep it civil, multiple developers here get "attacked" by you for having different opinions. It's ok to have different opinions even calls for action, and by aggressive reactions to someones opinions we even get stronger reactions and fortify them in the opinions we might not like :) 

Best wishes to Vuetify team and keep your good work, we all, by being developers,  know how it is to work under pressure and we should have empathy.

On the other side we also have our own dependent projects and our clients/bosses above our heads, and it would help both us and you I guess to be more transparent and objective regarding releases so that we can make our decisions on time and to avoid disappointments.

**The main info missing and being misleading as I see by reading community questions** is **on which date approximately or which release** we should expect that we have let's say **90% of the components/functionality provided in 2.x versions**

Thanks, best!

**Comment by quiquelhappy** [comment]
> Hmm, I was trying to keep bearing with Vuetify 3 (even implementing some workarounds on our ongoing projects) until target release is here.
> 
> Yet:
> 
> 
> 
> - official deadline for Vuetify Target release has passed (May) but we're still in Beta
> 
> - there are no answers from developers regarding not-working functionality even the one documented in Beta as working https://github.com/vuetifyjs/vuetify/discussions/15054
> 
> - we do not get updates when something is going to be delivered, after the self-defined deadlined has passed, so we can hardly make valid decisions
> 
> - as some of you mentioned here, it seems that the TARGET release of 3.0 (expected in the passed May) won't be a "real" target release in sense it will still lack too many components of their functionality, compared to 2.x version ...
> 
> - I would like to hear some opposing reassuring thoughts from developers, yet it seems that the waiting and workarounds inspired by "too optimistic" announcements by Vuetify developers is not justified anymore :(
> 
> - Especially that this is not ony waiting for Vuetify 3 but also preventing us from switching from Vue2 to Vue 3 and all the new ecosystem being built around it (so waiting on 2.7 and suggested (automatic) migration to 3.x using it is neither solution, esp. that it has also been prolonged for several months)
> 
> 
> 
> In another thread https://github.com/vuetifyjs/vuetify/discussions/14776 the [Quasar](https://quasar.dev/) is mentioned as an alternative while we wait for the Vuetify 3 reaching that level of stability and versatility. I guess I will give it a try.
> 
> 
> 
> P.S. Dear @MartinX3 please keep it civil, multiple developers here get "attacked" by you for having different opinions. It's ok to have different opinions even calls for action, and by aggressive reactions to someones opinions we even get stronger reactions and fortify them in the opinions we might not like :) 
> 
> 
> 
> Best wishes to Vuetify team and keep your good work, we all, by being developers,  know how it is to work under pressure and we should have empathy.
> 
> 
> 
> On the other side we also have our own dependent projects and our clients/bosses above our heads, and it would help both us and you I guess to be more transparent and objective regarding releases so that we can make our decisions on time and to avoid disappointments.
> 
> 
> 
> **The main info missing and being misleading as I see by reading community questions** is **on which date approximately or which release** we should expect that we have let's say **90% of the components/functionality provided in 2.x versions**
> 
> 
> 
> Thanks, best!

Absolutely agree

**Comment by KaelWD** [comment]
- They're estimates not deadlines.
- I don't have a link but that's been answered before, documentation is obviously one of the last things to be updated. 
- We don't know either.
- etc.

> 90% of the components/functionality provided in 2.x versions

I'd say we're pretty close, the only components missing are the big complex ones that depend on other functionality being stable:
- treeview ([in progress](https://github.com/vuetifyjs/vuetify/pull/14715))
- data table ([in progress](https://github.com/vuetifyjs/vuetify/tree/feat/v3-data-table))
- calendar ([in progress](https://github.com/vuetifyjs/vuetify/pull/15038))
- pickers ([in progress](https://github.com/vuetifyjs/vuetify/tree/feat/v3-date-picker))

**Comment by quiquelhappy** [comment]
> - They're estimates not deadlines.
> 
> - I don't have a link but that's been answered before, documentation is obviously one of the last things to be updated. 
> 
> - We don't know either.
> 
> - etc.
> 
> 
> 
> > 90% of the components/functionality provided in 2.x versions
> 
> 
> 
> I'd say we're pretty close, the only components missing are the big complex ones that depend on other functionality being stable:
> 
> - treeview ([in progress](https://github.com/vuetifyjs/vuetify/pull/14715))
> 
> - data table ([in progress](https://github.com/vuetifyjs/vuetify/tree/feat/v3-data-table))
> 
> - calendar ([in progress](https://github.com/vuetifyjs/vuetify/pull/15038))
> 
> - pickers ([in progress](https://github.com/vuetifyjs/vuetify/tree/feat/v3-date-picker))

Thanks for the update!

**Comment by sinisarudan** [comment]
thanks for the update @KaelWD , we're getting closer to some estimates.

Regarding DOCS, I'm not telling that the Docs are not updated.
On the contrary! The [v.3.0 docs for `v-select`](https://next.vuetifyjs.com/en/api/v-select/#props-items) state that `props-item` are working as expected.

Yet as I and other devs have stated, read [more here in the explained issue](https://github.com/vuetifyjs/vuetify/discussions/15054) - that's not the fact. I myself tried to test it even beyond docs in different ways..

The problem here is that we lose the point of reference/truth, on what can we rely in the current `beta.1` and the following versions of Vuetify?

Or you're trying to say that the docs DOES NOT reflect the currently released version and is written independently to reflect the final (desired) v.3.0?

If it does reflect, could anyone investigate that issue of v-select. As being one of the central UX components and not working as documented, I feel we are pushed for alternatives.

Thanks in andvance

**Comment by KaelWD** [comment]
You might've missed this
![Screenshot_20220602_213556](https://user-images.githubusercontent.com/16421948/171620794-aa64b58f-2add-4eaf-9c6d-e9ed577d71b1.png)
The example you linked to was for v2 and hasn't been updated yet, it needs `item-title="text"` to work with those objects

**Comment by AndreyYolkin** [comment]
> You might've missed this ![Screenshot_20220602_213556](https://user-images.githubusercontent.com/16421948/171620794-aa64b58f-2add-4eaf-9c6d-e9ed577d71b1.png) The example you linked to was for v2 and hasn't been updated yet, it needs `item-title="text"` to work with those objects

Do you mind receiving PRs with documentation updates?

**Comment by sinisarudan** [comment]
- I've seen that of course (actually every time I go through docs :) ) - yet I'm not worrying of is it outdated, but too advanced for the stated of the code it should document :)
- docs https://next.vuetifyjs.com/en/api/v-select/#props-item-props is for v3 and not v2 (I was using the proper doc)
- I was not trying any v2 example but was trying to make v3 v-select to work according to v3 [component doc](https://next.vuetifyjs.com/en/components/selects/), abd [api doc](https://next.vuetifyjs.com/en/api/v-select/#props-items).
- `item-title` prop is not documented, but **item-text, item-value and item-disabled** props. Nevertheless, neither is solving this issue.
![image](https://user-images.githubusercontent.com/1769627/171724295-81666a67-62d8-4eec-b555-e8f0ce7f6ad4.png)
- So, after trying all possible and impossible variants to handle `v-select` supporting objects instead of simple text, I can confirm it's not working. ANd as I've said, you have other dev(s) claiming the same.
- and that is not some advanced functionality, but needed for almost any app.
- if we accept that there is some Docs misbehaviour, the point is still: What is our point of reference then, when we use components and their props? or even for community-testing of the Beta versions to give you any feedback?

**Comment by sinisarudan** [comment]
can we get any answer instead of marking the comment as outdated??
I would really like to keep giving chance to Vuetify, but by this kind of communication it's even harder to feel reliability of the framework.
Thanks

**Comment by genofire** [comment]
Please keep the issue uptodate and add e.g. v-calender 
which is makes by @KaelWD (thx):
https://github.com/vuetifyjs/vuetify/pull/15038

**Comment by johnleider** [comment]
> can we get any answer instead of marking the comment as outdated??
> 
> I would really like to keep giving chance to Vuetify, but by this kind of communication it's even harder to feel reliability of the framework.
> 
> Thanks
> 
> 

Reliability of the framework? Give me a break. Take this convo to our discord and stop whining in this thread.

**Comment by johnleider** [comment]
> @johnleider Please keep the issue uptodate and add e.g. v-calender which is makes by @KaelWD (thx): #15038

It is up to date. `v-calendar` is not shipping with v3.0.

**Comment by MatthewAry** [comment]
Here is my unsolicited feedback on this recent tangent, I think others might find it useful. Note: I am making assumptions about the state of things in this project so if I am wrong please feel free to correct me.

@sinisarudan Vuetify 3 is beta, which usually means:

- Development has priority over authoring docs right now. 
- Errata, and missing information is to be expected in the docs while in beta up until V3 reaches some unspecified level of maturity.
- The development team has no obligation to provide API guarantees until GA.
- The scope of the V3 initial release is locked in, what you see on the release checklist is meant to give you an idea how close/far they are to release.
- I have my own questions about V3 but I know that it would be inappropriate to ask them HOW to do something with V3 Beta, and more appropriate to ask them if a specific feature, or component will be included in the initial release (excluding anything that they may consider obvious)
- The dev team is free to ignore discussions or bug reports until V3 is released. They are more likely to interact with you if you intend to contribute code or if you engage with them on something that team members are currently concerned with or occupied with.

I specifically posted to this task, because I wanted to know about a feature. I didn't get a response, and it's no big deal for me right now. I have other things to worry about.

TLDR: Don't expect any kind of support from the dev team for non GA releases. They're still making V3 and will more likely converse with you as long as the discussion has to do with any of their present concerns.

**Comment by websitevirtuoso** [comment]
I think that beta is an incorrect tag for the current situation. It should be alpha. So many components doesn't work properly like they worked in 2.6 version. Everything great. Let's just wait. This is great UI framework. I am still waiting my v-data-table component

**Comment by UkonnRa** [comment]
> > I think that beta is an incorrect tag for the current situation. It should be alpha. So many components doesn't work properly like they worked in 2.6 version. Everything great. Let's just wait. This is great UI framework. I am still waiting my v-data-table component
> 
> Which components do not work properly?

The Autocomplete in doc is still broken.

![autocomplete](https://user-images.githubusercontent.com/16066720/171966231-df3e2ca7-8698-4d7e-94a3-e6b5dcacbb3a.png)

**Comment by johnleider** [comment]
I feel like I shouldn't have to say this since it is plastered on every page of the next docs, but alas:

![image](https://user-images.githubusercontent.com/9064066/171969413-4c89abf3-1b19-448d-a21e-34711979eb7a.png)

**Comment by websitevirtuoso** [comment]
> > I think that beta is an incorrect tag for the current situation. It should be alpha. So many components doesn't work properly like they worked in 2.6 version. Everything great. Let's just wait. This is great UI framework. I am still waiting my v-data-table component
> 
> Which components do not work properly?

Vuetify 2.6
![image](https://user-images.githubusercontent.com/5112918/171979077-ba7ef7b9-5f96-4fa1-8e0c-2228b1f4a482.png)

Vuetify 3
![image](https://user-images.githubusercontent.com/5112918/171979087-d39d9206-bd94-48a9-8e5e-c22a7f5b9165.png)


Vuetify3 beta3
Component breadcrumbs. Has invalid styles and html structure
![image](https://user-images.githubusercontent.com/5112918/171979191-c0906edb-6461-4dac-b7c1-465308ddb278.png)


Vuetify3 beta3.
Validation Error should have red color but it is gray.
In beta2. It was red
![image](https://user-images.githubusercontent.com/5112918/171979212-2cc95267-112e-4aee-a0db-58081d7e77f0.png)

I sm uding vite and latest version of everything. let me know, I can provide more info. Very appreciate for everything

**Comment by MartinX3** [comment]
Please learn how to read and stop spamming the `Vuetify 3.0 Release CHECKLIST` issue ticket.

I didn't subscribe it to read wining people screaming for ETA without really helping the project and ignoring text warnings.

**Comment by johnleider** [comment]
> Would like to suggest adding a Nuxt3 plugin to this list as well.

I will need help with this as I do not use NUXT. Happy to help however I can though.
